infra/helm/templates/api-gateway-deployment.yaml

```
@@ -4,47 +4,72 @@ metadata:
   labels:
     io.kompose.service: {{ .Values.apiGateway.name }}
   name: {{ .Values.apiGateway.name }}
+  namespace: {{ .Values.namespace }}
+    io.kompose.service: {{ .Values.apiGateway.name }}
+  name: {{ .Values.apiGateway.name }}
```

YAMLlint (1.37.1)

[error] duplication of key "name" in mapping

(key-duplicates)